### PR TITLE
Update vet clinic

### PR DESCRIPTION
**Pull request by luccamicale**
## Update Vet Clinic Database 💯 

##https://github.com/luccamicale/vet_clinic 🔢 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I used the update
- [x] I used the delete
- [x] I used the custom queries 🥇 💯 👍 

DATA3
DATA4
DATA5
DATA6
COSTUM QUERYS

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR
```
Comment (zieeco) [inline schema.sql]: - Hey, please, let's specify the numbers of characters the column can take to help maintain our database to improve its performance. Not specifying means the column can take any number of characters and this is not ideal for your database. eg `varchar(100);`

File: data.sql [hunk 2]
```diff
@@ -3,4 +3,14 @@
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', 'February 3, 2020', 0, true, 10.23);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', 'November 15, 2018', 2, true, 8);
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 'January 7, 2021', 1, false, 15.04);
-INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, true, 11);
\ No newline at end of file
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, true, 11);
+
+/* Update animals */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', 'Febrary 8, 2020', 0, false, 11);
```
Comment (zieeco) [inline data.sql]: - Hey, please, let's kindly enure we pass the correct value for the datatype to help reduce issues one could face with not sending the correct data to the correct datatype in the database. The value for the weight should be a floating number as it is a decimal and not an integer. Kindly update the 11 to `-11.0`

File: data.sql [hunk 3]
```diff
@@ -3,4 +3,14 @@
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', 'November 15, 2021', 2, true, 5.7);
```
Comment (zieeco) [inline data.sql]: - Hey, please, kindly ensure you've sent the correct value to the database as this should be `-5.7`

File: data.sql [hunk 4]
```diff
@@ -3,4 +3,14 @@
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', 'April 2, 1993', 3, false, 12.13);
```
Comment (zieeco) [inline data.sql]: - Hey, please, kindly ensure you've sent the correct value to the database as this should be `-12.13`

File: data.sql [hunk 5]
```diff
@@ -3,4 +3,14 @@
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', 'June 12, 2005', 1, true, 45);
```
Comment (zieeco) [inline data.sql]: - Hey, please, kindly ensure you've sent the correct value to the database as this should be `-45.0`

File: data.sql [hunk 6]
```diff
@@ -3,4 +3,14 @@
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', 'June 7, 2005', 7, true, 20.4);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', 'October 13, 1998', 3, true, 17);
```
Comment (zieeco) [inline data.sql]: - Hey, please, let's kindly ensure we pass the correct value for the datatype to help reduce issues one could face with not sending the correct data to the correct datatype in the database. The value for the weight should be a floating number as it is a decimal and not an integer. Kindly update the 11 to `17.0`

File: data.sql [hunk 7]
```diff
@@ -3,4 +3,14 @@
...
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', 'May 14, 2022', 4, true, 22);
```
Comment (zieeco) [inline data.sql]: - Hey, please, let's kindly enure we pass the correct value for the datatype to help reduce issues one could face with not sending the correct data to the correct datatype in the database. The value for the weight should be a floating number as it is a decimal and not an integer. Kindly update the 11 to `22.0`

File: schema.sql (hunk 1 above)
Comment (zieeco) [reply schema.sql]: #### README FILE

- Hey, great job adding the readme file, however, let's kindly update our readme to the current readme [template](https://github.com/microverseinc/readme-template) to conform to the new structure and be consistent to make our readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @luccamicale ,


Great job so far! 👏 
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice querry. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I modified the problem with the data.sql. Now My project is complete and should be approved!!! 💯

**Comment by luccamicale** [comment]
and i added the professional Readme

File: schema.sql [hunk 8]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(100)
```
Comment (Reem-lab) [inline schema.sql:10]: - [ OPTIONAL ] I think this is not the right way to add a colmun you should use the alter command so you can add a column to an existed table so kindly use the alter command for adding a new column. ✅🙏🏻

**Review by Reem-lab — APPROVED** [review]
# Hi @luccamicale 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- Followed Gitflow ✅
- Great working on the functionality 🔥🚀🎉
- Descriptive PR ✅
- Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**